tests/integration/mounts.bats

```diff
@@ -19,3 +19,12 @@ function teardown() {
 	[ "$status" -eq 0 ]
 	[[ "${lines[0]}" == *'/tmp/bind/config.json'* ]]
 }
+
+@test "runc run [bind mount w/o explicit \"bind\" opt but w/ explicit \"bind\" type]" {
+	update_config 	' .mounts += [{"source": ".", "destination": "/tmp/bind", "type":"bind"}]
+			| .process.args |= ["ls", "/tmp/bind/config.json"]'
+
+	runc run test_bind_mount
+	[ "$status" -eq 0 ]
+	[[ "${lines[0]}" == *'/tmp/bind/config.json'* ]]
```

Comment:
this will also match something like "ls: /tmp/bind/config.json: no such file or directory" which is not good.

I think we should list the directory instead, and check the file is present in the output.

Reply:
We check the status code, so it won't match `ls: /tmp/bind/config.json: no such file or directory`

Reply:
well in this case it does not make sense to check the output, since `ls` will definitely return the error if the file is not there, so this check is redundant

Reply:
So I guess either
 - list the directory, check the exit code, check the output to contain file name
or
 - list the file, check the exit code